src/assets/sass/protocol/components/_emphasis-box.scss

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../includes/lib';
+
+/* -------------------------------------------------------------------------- */
+// Base emphasis Box class
+section.mzp-c-box-emphasis-medium {
+    background: $color-white;
+    border-radius: 6px;
+    display: inline-block;
+    text-align: center;
+    margin: $spacing-md;
+}
+
+.mzp-c-box-emphasis-content {
+    text-align: center;
+    padding: $spacing-xl;
+}
+
+.mzp-c-box-emphasis-fuzzy {
+    background: $color-white;
+    box-shadow: 0 0 15px 0 $color-gray-30;
+    border-radius: 6px;
+}
+
+.mzp-c-box-emphasis {
+    background: $color-white;
+    box-shadow: 0 0 10px 0 $color-gray-30;
```

Comment:
We should use a transparent shadow so it will also work on colored backgrounds. This could either be `rgba(0, 0, 0, .25)` (or something like that, not sure of the correct alpha value) or you can use a Sass function like `transparentize($color-black, .75)`.